apps/prairielearn/src/sync/course-db.ts

```diff
@@ -74,7 +83,11 @@ const DEFAULT_ASSESSMENT_SETS = [
   { abbreviation: 'U', name: 'Unknown', heading: 'Unknown', color: 'red3' },
 ];
 
-const DEFAULT_TAGS = [
+const DEFAULT_TAGS: {
+  name: string;
+  color: Color;
+  description?: string;
+}[] = [
```

Comment:
`schemas/infoCourse.ts` has a `TopicSchema`/`Topic` exported. Should we do the same with tags here? You did that above with `DEFAULT_ASSESSMENT_SETS`.

Reply:
Resolved in 604d41b362247928a520fb640163942c0df6d456.